main/auth/gae.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+from __future__ import absolute_import
```

Comment:
In this case it is necessary for correct import
``` python
from google.appengine.api import users
```
Because we import ```gae.py``` (in ```__init__.py```) through relative import
``` python
from .gae import *
```